portfolio/src/main/webapp/index.html

```diff
@@ -19,102 +19,155 @@ <h1 class="page_headers">Ricardo Chapa's Portfolio</h1>
+                <input type="submit" alt="Click to submit your comment" id="comment_submit"/>
+              </form>
+            </div>
+            <br/><br/>
+            <div id="comment-display-container"></div>
+            <!-- Comment Header Template -->
+            <template>
+              <div id="user-comment-container">
+                <div id="user-comment-header">
+                  <div id="comment-header-wrapper">
+                    <b class="username"></b>
+                  </div>
+                </div>
+                <p></p>
+                <div id="comment-header-wrapper">
+                  <div id="close-button-wrapper">
+                    <i class= "single-fa fa fa-remove"></i>
+                  </div>
+                </div>
+              </div>
+            </template>
+            <div id="comment-selector-container">
+              <form action="/load-comments" method="GET">
+                <label for="number-of-comments"> Number of Comments: </label>
+                <input type="number" name="number-of-comments" id="number-of-comments" max="7" min="0" value="0"/>
+                <br/><br/>
+                <input 
+                  type="button" 
+                  id="comment_submit" 
+                  onclick="loadComments()" 
+                  onsubmit="return false"
+                  value="Load comments"
+                />
+              </form>
             </div>
           </div>
         </div>
-        <!-- Gallery Container -->
-        <div id="my_gallery">
-          <h2 class="page_headers">My Gallery</h2>
-          <div id="paragraph_container">
-            <p class="justify_text">
-              One hobby that I ocassionally have is taking pictures. 
-              Even thought I'm not much of a photographer, 
-              I really like to capture the moment of any special time that I'm having in my life,
-              just for when I see that picture I can remember the feeling that I was having in that moment.
-            </p>
-            <button class = "random_picture_button" onclick="onClickImageCyclerButton()">Click to start the random image cycler</button>
-            <!-- Random Image Container -->
-            <div id="randomPicture_and_description_container">
-              <img id="random_picture_container"></img>
-              <div id="picture_description_container"></div>
+        <div id="content">
+          <!-- Portafolio Container -->
+          <div id="information_container">
+            <!-- Image and Description Container -->
+            <div id="image_and_description">
+              <h2 class="page_headers">Profile</h2>
+              <img 
+                class="my_picture" 
+                src="/../images/profile_picture.jpg" 
+                alt="Picture taken in the library at Tecnologico de Monterrey of me, 
+                    smiling and wearing a hoodie with the RoBorregos logo in it (A robotic Lamb).">
+              </img>
```

Comment:
I might be wrong, but I think the `<img>` element does not have a closing tag.

Reply:
In theory it can work without it but if you put it there it works as well